members.json

```
@@ -1402,6 +1402,7 @@
 				"start": "2025-07-01",
 				"end": "2026-06-30",
 				"type": "elected",
+				"chair": "true",				
```

`chair` is a boolean so it shouldn't the quotes around the value:
```suggestion
				"chair": true,
```
After you merged that PR, I'll update the other values.